src/components/Diff/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React, { useMemo } from 'react';
+import { object, oneOf } from 'prop-types';
```

Comment:
I see this file is getting renamed, but is still only used for diffing Rules. Similarly, `JsDiff` is only used for diffing Releases. I've got no objection to them being generic, but I think they could use better names. It's not clear to me what the difference between them is based on the current names.